packages/components/src/Dropdown/Dropdown.module.css

```diff
@@ -2,6 +2,14 @@
   @apply relative bg-white;
 }
 
+.compact {
+  @apply w-min;
+}
+
+.compact > button {
+  @apply p-1;
+}
```

Comment:
Maybe we could thread the `compact` prop through `DropdownButton` as well? It's a shared component between other things that use this "dropdown style" (like Popovers)

Reply:
Instead of adding variant for DropdownButton, I just styled it on `DropdownTrigger`. Let me know if you think otherwise.

Reply:
I'm also wondering about whether it should be a prop, but I'm not super opinionated about it. I'm fine with waiting for a `Popover` to need a `compact` button before refactoring this.